src/test/end.t.sol

```diff
@@ -191,6 +191,7 @@ contract EndTest is DSTest {
         gov.setOwner(address(flop));
 
         vow = new Vow(address(vat), address(flap), address(flop));
+        flap.rely(address(vow));
```

Comment:
This can be removed, this same permission was already set at the end of the `setUp` function.